app/history/page.tsx

```diff
@@ -0,0 +1,241 @@
+import btc from "../../public/coin-logos/btc-logo.png"
```

Comment:
snyc your fork and make use of the svg versions of the coins. If a particular svg coin is not available in the repo, you can go ahead and export it from the figma file and make use of it.